testharness/build.gradle

```diff
@@ -3,13 +3,7 @@ plugins {
 }
 
 dependencies {
-    implementation (project(":library")) {
-        // the msal4j transitive dependency is still pulling in an old version of json-smart
-        // which is vulnerable to CVE-2023-1370
-        // see https://github.com/AzureAD/microsoft-authentication-library-for-java/issues/611
-        exclude group: 'net.minidev', module: 'json-smart'
-    }
-    implementation 'net.minidev:json-smart:2.5.0'

```

Comment:
I am no longer seeing this vulnerability per Gradle dependencyInsight but will confirm via Veracode feature branch scan.